.node-version

```diff
@@ -0,0 +1 @@
+20.11.0
```

Comment:
This is used to force the Node version on Netlify. I'm assuming it's currently configured in the UI itself and not from code.